Lombiq.NodeJs.Extensions/scripts/validate-config.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @summary Validates the Node.js Extensions configuration object.
```

Comment:
Not important for this PR, but isn't this basically schema validation? We should consider replacing the custom logic with existing tools. This looks suitable: https://www.npmjs.com/package/jsonschema If you agree I'll create a followup issue.

Reply:
I completely agree, but I wanted to avoid scope explosion. Go right ahead!

Reply:
I've created the issue: https://github.com/Lombiq/NodeJs-Extensions/issues/15

I'm all for standing up to scope explosion. We should be more willing to create followup issues instead of bloating existing ones.